snmp/fail2ban

```diff
@@ -0,0 +1,141 @@
+#!/usr/local/bin/perl
+
+# Add this to your snmpd.conf as below.
+# extend fail2ban /etc/snmp/fail2ban
+#
+# Then add to your cron tab...
+# */3 * * * * /root/snmp-extends/fail2ban.pl -u
+
+#make sure this path is correct
+my $f2bc="/usr/local/bin/fail2ban-client";
```

Comment:
Try to make this work for as many users as possible.  Use just `fail2ban-client` if you can.